at_lookup/test/secondary_address_cache_test.dart

```diff
@@ -101,5 +105,11 @@ void main() async {
       expect(cache.getCacheExpiryTime(atSign), isNotNull);
       expect((approxExpiry - cache.getCacheExpiryTime(atSign)!) < 100, true);
     });
+
+    test('test update cache for atsign which is not yet cached', () async {
+      var atSign = 'notCachedAtSign1';
```

Comment:
maybe add an assertion that the atsign is not cached before calling cache.getAddress

Reply:
done